.github/workflows/nightly-tests.yml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2024 Agnostiq Inc.
+#
+# This file is part of Covalent.
+#
+# Licensed under the Apache License 2.0 (the "License"). A copy of the
+# License may be obtained with this software package or at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Use of this file is prohibited except in compliance with the License.
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: nightly-tests
+
+on:
+  push:
+    branches:
+      - develop
+  schedule:
+    - cron: "*/10 * * * *"
```

Comment:
this will run it every 10 minutes

Reply:
Yeah, I wanna test it once it gets merged. I'll manually disable the workflow and then afterwards when its working fine, I'll change this to per day.

Reply:
Updated to `0 0 * * *`